decidim-admin/app/controllers/decidim/admin/participatory_process_step_ordering_controller.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+require_dependency "decidim/admin/application_controller"
+
+module Decidim
+  module Admin
+    # Controller that allows managing all the Admins.
+    #
+    class ParticipatoryProcessStepOrderingController < ApplicationController
+      def create
+        authorize ParticipatoryProcessStep
+
+        ReorderParticipatoryProcessSteps.call(collection, params[:items_ids]) do
+          on(:invalid) do
```

Comment:
See previous discussion in https://github.com/codegram/decidim/pull/137#discussion_r84491443